### PR TITLE
omake 0.10.3 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Ressurected 8 years later, now that it's being [maintained again](https://github.com/ocaml-omake/omake)!  Last seen in Homebrew/homebrew-core@a2d3be19ffe6ed8150ec5f3c56aa4e24bc2a2697 where it was “graveyarded.”

### CODE REVIEW

**Comment by pthariensflame** [comment]
Failure is due to notability requirements.  Hopefully we can bypass those since this is a resurrection rather than a truly _new_ formula.

File: Formula/omake.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  conflicts_with "etsh", :because => "both install 'osh' binaries"
+
+  def install
+    system "./configure", "-prefix", prefix
+    system "make", "install"
+  end
+
+  test do
+    # example run adapted from the documentation's "quickstart guide"
+    mkdir "test_project" do
```
Comment (jonchang) [inline Formula/omake.rb:21]: Tests already run in their own directory so this shouldn't be necessary unless it's required by the software in question.

File: Formula/omake.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      system bin/"omake", "--install"
+      "hello_code.c".write <<~EOF
```
Comment (jonchang) [inline Formula/omake.rb:23]:
```suggestion
      (testpath/"hello_code.c").write <<~EOF
```

(assuming the `mkdir` above is removed)

File: Formula/omake.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        #include <stdio.h>
+
+        int main(int argc, char **argv)
+        {
+            printf("Hello, world!\n");
+            return 0;
+        }
+      EOF
+      "OMakefile".write <<~EOF
```
Comment (jonchang) [inline Formula/omake.rb:32]:
```suggestion
      (testpath/"OMakefile").write <<~EOF
```

**Review by jonchang — CHANGES_REQUESTED** [review]
Nice work, some suggestions to fix the test error.

File: Formula/omake.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  test do
+    # example run adapted from the documentation's "quickstart guide"
+    system bin/"omake", "--install"
+    (testpath/"hello_code.c").write <<~EOF
+      #include <stdio.h>
+
+      int main(int argc, char **argv)
+      {
+          printf("Hello, world!\n");
```
Comment (jonchang) [inline Formula/omake.rb:28]:
```suggestion
          printf("Hello, world!\\n");
```

**Comment by chenrui333** [comment]
> Ressurected 8 years later, now that it's being maintained again! Last seen in a2d3be1 where it was “graveyarded.”

🎉  Nice finding.

**Pull request closed without merging**